ruby-audit-logs-example/app.rb

```diff
@@ -22,77 +23,118 @@
 )
 
 get '/' do
+  before = params[:before]
+  after = params[:after]
+  if !before
+    @organizations = WorkOS::Organizations.list_organizations(
+      limit: 5
+    )
+  else
+    @organizations = WorkOS::Organizations.list_organizations(
+      limit: 5,
+      before: before,
+      after: after
+    )
+  end
+  @before = @organizations.list_metadata["before"]
+  @after = @organizations.list_metadata["after"]
   erb :login, :layout => :layout
 end
 
-post '/set_org' do
-  @organization_id = params[:org]
-
-  session[:organization_id] = @organization_id
-  
-  organization = WorkOS::Organizations.get_organization(
-  id: @organization_id
-  )
-  
-  @org_name = organization.name
-  session[:organization_name] = @org_name
-  erb :send_events, :layout => :layout
-end
 
 get '/set_org' do
-  @organization_id = session[:organization_id]
-  @org_name = session[:organization_name]
+  @organization = WorkOS::Organizations.get_organization(
+    id: params[:id]
+  )
+  @today_iso = Time.now.utc.iso8601
+  @last_month_iso = (Time.now - (30 * 86400)).utc.iso8601
   erb :send_events, :layout => :layout
 end  
 
-post '/send_event' do
-  event_type = params[:event]
-  @organization_id = session[:organization_id]
-  @org_name = session[:organization_name]
-
-  events = [
-        $user_signed_in,
-        $user_logged_out,
-        $user_organization_deleted,
-        $user_connection_deleted,
-    ]
-
-  event = events[event_type.to_i]
+get '/events' do
+  link = WorkOS::Portal.generate_link(
+    organization: params[:organization_id],
+    intent: params[:intent],
+    )
+  redirect link
```

Comment:
After sending a successful event I got redirected back to the home page where you pick an organization. Can we redirect back to the organization's page you were just on after you send an event?

Reply:
![CleanShot 2023-03-27 at 14 22 29](https://user-images.githubusercontent.com/47123183/228057665-689f6de2-765e-437a-88c1-9c127f7c07ed.gif)

Reply:
done